pkg/invoice/response.go

```diff
@@ -0,0 +1,29 @@
+package invoice
+
+import "time"
+
+type Response struct {
+	CurrencyID        string     `json:"currency_id"`
+	DateCreated       *time.Time `json:"date_created"`
+	DebitDate         *time.Time `json:"debit_date"`
+	ExternalReference string     `json:"external_reference"`
+	ID                int        `json:"id"`
+	LastModified      *time.Time `json:"last_modified"`
+	NextRetryDate     *time.Time `json:"next_retry_date"`
+	Payment           Payment    `json:"payment"`
+	PaymentMethodID   string     `json:"payment_method_id"`
+	PreApprovalID     string     `json:"preapproval_id"`
+	Reason            string     `json:"reason"`
+	RetryAttempt      int        `json:"retry_attempt"`
+	Status            string     `json:"status"`
+	Summarized        string     `json:"summarized"`
+	TransactionAmount float64    `json:"transaction_amount"`
+	Type              string     `json:"type"`
+}
+
+// InvoicePayment contains information about payment
+type Payment struct {
```

Comment:
estamos colocando o sufixo response
```suggestion
type PaymentResponse struct {
```